Source/Extensions/Blazorise.Exporters/IExportResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Blazorise.Exporters;
+
+/// <summary>
+/// Represents a generic result of an export operation, indicating whether the export was successful.
+/// </summary>
+public interface IExportResult
+{
+    /// <summary>
+    /// Indicates whether an operation was successful.
+    /// </summary>
+    bool Success { get; init; }
```

Comment:
Should we also have `Errors` if `Success == false`?

Reply:
Error handling isn't quite there yet.
Yeah, somehow yes.
It has to start with the js functions - for example when the clipboard isn't available or saving file fails.

Reply:
Should I improve on the error handling now?